lib/FormCustomProperties/CustomPropertiesListField.js

```diff
@@ -141,11 +141,19 @@ export default class CustomPropertiesListField extends React.Component {
       const { note, publicNote, value } = val ? val[0] : {};
 
       if (!customProperty.primary && customProperty.value && !value) {
```

Comment:
Will this be true and throw an error when the valid number/decimal `value` is `0`?

Reply:
See comment here -> https://github.com/folio-org/ui-licenses/pull/320/files#r393036097